tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +31,11 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(items) == 0
+    assert item_a not in items
+    assert item_b not in items
+    assert item_c not in items
```

Comment:
We only need the first assertion in this case. If the length of `items` is `0` then it is empty, we know `item_a`, `item_b`, and `item_c` are not in it.